src/main/java/uk/gov/justice/probation/courtcasematcher/restclient/model/courtcaseservice/CCSExtendedCase.java

```diff
@@ -43,6 +45,7 @@ public static CCSExtendedCase of(CourtCase courtCase) {
     public CourtCase asDomain() {
         return CourtCase.builder()
                 .caseId(caseId)
+                .hearingId(hearingId)
```

Comment:
The unit test for this relies on the value returned from `DomainDataHelper.aCourtCaseWithAllFields();` - it doesn't look like hearingId has been added to that though so that'll need doing or it won't get taken into account.